docs/TFE_links.md

```diff
@@ -0,0 +1,30 @@
+# Timelapse feature explorer links for data shown in figures
+
+[Timelapse Feature Explorer](https://timelapse.allencell.org/) is a web-based tool that allows users to view timelapse segmentations with interactive and customizable colorization mapped according to relative intensities of quantitative features. In this viewer, nuclear segmentations from each individual colony timelapse are colorized by a chosen feature, with each nucleus in each frame being colored by its feature value in that frame. The resulting timelapses demonstrate how growth and shape features vary for all nuclei across the colony and over time, allowing for a qualitative assessment of spatiotemoral variation of these features. The tool also allows for selection of single nuclear trajectories, to visualize how individuals behave over time and compare to the population. 
```

Comment:
Nit: What do you think of "relative magnitudes of quantitative features"? Not sure intensities is quiiiiiiiite right.